include/sys/dmu_objset.h

```diff
@@ -235,7 +235,8 @@ void dmu_objset_evict_dbufs(objset_t *os);
 inode_timespec_t dmu_objset_snap_cmtime(objset_t *os);
 
 /* called from dsl */
-void dmu_objset_sync(objset_t *os, zio_t *zio, dmu_tx_t *tx);
+zio_t *dmu_objset_sync(objset_t *os, zio_t *zio, dmu_tx_t *tx);
+void dmu_objset_sync_sublists_done(zio_t *zio);
```

Comment:
I think this can be removed from the header and made `static`.

Reply:
removed, thanks.